app/wama.web/Controllers/CheckInController.cs

```diff
@@ -26,10 +26,10 @@ public IActionResult Index(string memberId)
             if (ModelState.IsValid && !string.IsNullOrWhiteSpace(memberId))
             {
                 var loginCredential = _CheckInService.GetLogInCredential(memberId);
-
                 if (loginCredential == null)
                 {
-                    SetErrorMessages("The ID that you entered does not exit.Please Try again or contact the manager ");
+                    ViewBag.IsNull = "yes";
```

Comment:
Nice approach, but this could be done in a bit cleaner fashion.

What about passing some sort of value in the ViewBag or ViewData